src/kemal/base/builder.cr

```diff
@@ -0,0 +1,108 @@
+class Kemal::Base
+  module Builder
+    getter custom_handlers = [] of Tuple(Nil | Int32, HTTP::Handler)
+    getter filter_handlers = [] of HTTP::Handler
+    @handler_position = 0
+
+    def clear
+      @router_included = false
+      @handler_position = 0
+      @default_handlers_setup = false
+
+      handlers.clear
+      custom_handlers.clear
+      filter_handlers.clear
+      error_handlers.clear
+
+      route_handler.clear
+      websocket_handler.clear
+    end
+
+    def handlers=(handlers : Array(HTTP::Handler))
```

Comment:
I supposed this is the same `Kemal.config.handlers =` which gives the ability to totally replace the handlers.